source/_patterns/00-atoms/forms/select.yaml

```diff
@@ -55,3 +62,4 @@ schema:
     - options
   dependencies:
     label: [id]
+    messageId: [isInvalid]
```

Comment:
Other way around? `isInvalid` requires `messageId`?

Reply:
Could be bidirectional? `messageId` is superfluous is there's no `isInvalid`.

Reply:
The intent is that if `isInvalid` is present (it can only be true if it is), then there must be a `messageId`.

Reply:
`isInvalid: [messageId]`